assets/scripts/photo/ui.js

```diff
@@ -6,26 +6,26 @@ const store = require('../store')
 const events = require('./events')
 
 const onCreatePhotoSuccess = function (data) {
-  $('#nav-message').text('Create Photo Success')
+  $('#upload-message').text('Create Photo Success').css('color', 'green')
   $('#photo-create').trigger('reset')
 }
 
 const onCreatePhotoFailure = function (data) {
-  $('#nav-message').text('Create Photo Failure')
+  $('#upload-message').text('Create Photo Failure')
   $('#photo-create').trigger('reset')
 }
 
 const onIndexPhotosSuccess = function (data) {
-  // $('#nav-message').text('Received photos')
   const indexPhotosHTML = indexPhotosTemplate({ photos: data.photos })
   $('#index-wrapper').html(indexPhotosHTML)
 }
 
 const onIndexPhotosFailure = function (data) {
-  $('#nav-message').text('Create Photo Failure')
+  $('#index-message').text('Create Photo Failure').css('color', 'red')
 }
 
 const onShowPhotoSuccess = function (data) {
+  $('.message').text('')
```

Comment:
Are we sure we want to clear the text of anything with class `message`? Might be better to clear whatever you're specifically concerned about showing.